src/backend/tests/kernelprog/vecadd/vecadd.driver.sexp

```diff
@@ -46,26 +64,26 @@
         (declare i int)
 
         (declare devCount int)
-        (declare device ,cu_device)
-        (declare context ,cu_context)
-        (declare module ,cu_module)
-        (declare kernel_func ,cu_function)
+        (declare device ,CUdevice)
+        (declare context ,CUcontext)
+        (declare module ,CUmodule)
+        (declare kernel_func ,CUfunction)
 
         ;; CUDA initialization and context creation
         ,(error_check (call cuInit 0))
         ,(error_check (call cuDeviceGetCount (ptr-to devCount)))
         ,(error_check (call cuDeviceGet (ptr-to device) 0))
-        ,(error_check (call cuCtxCreate (ptr-to context) 0 device))
+        ,(error_check (call cuCtxCreate_v2 (ptr-to context) 0 device))
```

Comment:
why does this have to `_v2`?

Reply:
`cuCtxCreate` happens to be a macro. And macros won't show up in the JSON AST :/

Reply:
Okay, the issue is a little more complicated than that. It's defined both as a macro and as a function:
```C
// ...
#define cuCtxCreate                         cuCtxCreate_v2
// ...
CUresult CUDAAPI cuCtxCreate(CUcontext *pctx, unsigned int flags, CUdevice dev);
// ...
```
As far as I can see, neither definition is within a conditional (e.g. `#if`, `#ifdef`) block.

Reply:
Look up zig and see how it parses it: https://ziglang.org/learn/overview/#integration-with-c-libraries-without-ffibindings